src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -126,12 +127,13 @@ void HexagonGame::initLua_MainTimeline()
 void HexagonGame::initLua_EventTimeline()
 {
     lua.writeVariable("e_eventStopTime", [this](float mDuration) {
-        eventTimeline.append<Do>([=, this] { status.timeStop = mDuration; });
+        eventTimeline.append<Do>(
+            [=, this] { status.pauseTime(ssvu::getSecondsToFT(mDuration)); });
     });
 
     lua.writeVariable("e_eventStopTimeS", [this](float mDuration) {
         eventTimeline.append<Do>(
-            [=, this] { status.timeStop = ssvu::getSecondsToFT(mDuration); });
+            [=, this] { status.pauseTime(ssvu::getSecondsToFT(mDuration)); });
```

Comment:
I mean since your pause time code is already written in seconds, you can get rid of this conversion entirely.